website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -72,6 +72,15 @@ by the HSM and one backed by Vault -- can satisfy both use cases. Operators
 can make roles setting maximum TTLs for each issuer and consumers of the
 mount can decide which to use.
 
+### Always Configure a Default Issuer
+
+For backwards compatibility, [the default issuer](/api-docs/secret/pki#read-issuers-configuration)
+is used to service the existing PKI endpoints. When certificates are revoked
```

Comment:
This language assumes a past and present.  "existing" won't mean much to newcomers.  Instead I'd phrase this as the endpoints with no issuer or similar.

Reply:
Good point, thanks!